src/bundle/Resources/public/js/scripts/admin.location.view.js

```diff
@@ -113,6 +113,13 @@
             return total;
         }, {});
         const udwConfigBulkMoveItems = JSON.parse(container.dataset.udwConfigBulkMoveItems);
+        const mfuContentTypesMap = Object.values(eZ.adminUiConfig.contentTypes).reduce((contentTypeDataMap, contentTypeGroup) => {
```

Comment:
We need to pass different config to MFU because the one based on: `container.dataset.contentTypes` contains only CTs of elements displayed in Sub-items. 

Given that `mfuContentTypesMap` in contrast to  `contentTypesMap` (which is passed to Sub-items) does not contain whole CT objects but rather data stored in `eZ.adminUiConfig.contentTypes`, are we OK with passing `mfuContentTypesMap` to MFU as `contentTypesMap` - ping @dew326  @sunpietro ?

Reply:
I think the approach is ok.

Reply:
Is a difference only in missing data or in the data structure as well?

Reply:
Thanks for explanation. Can't the backend provide this structure instead?

Reply:
We opted for this solution mostly because this is more efficient than extracting objects for all Content Types. But maybe it is way to go. This question is more to @adamwojs .